src/main/java/mekanism/client/render/hud/MekaSuitHUD.java

```diff
@@ -66,6 +76,17 @@ public void render(ForgeIngameGui gui, PoseStack poseStack, float partialTicks,
         }
     }
 
+    private int makeComponent(Consumer<List<Component>> adder, List<List<Component>> initial) {
+        List<Component> list = new ArrayList<>();
+        adder.accept(list);
+        int size = list.size();
+        if (size > 0) {
+            initial.add(list);
+            return size;
```

Comment:
Remove this extra return statement and just change the `return 0;` at the bottom to `return size;`